poe-tasks/manifest-only-connector-tasks.toml

```diff
@@ -44,6 +44,8 @@ else
 fi
 '''
 test-integration-tests = "airbyte-cdk connector test ${POE_PWD}"
+# Identical to test-integration-tests except excludes tests marked with @pytest.mark.requires_creds
+test-integration-tests-no-creds = "airbyte-cdk connector test ${POE_PWD} --pytest-arg='-m' --pytest-arg='not requires_creds'"
```

Comment:
Good news! The airbyte-cdk CLI now includes a --no-creds CLI arg, as of 6.60.10. You can update this syntax appropriately. (Might take a minute to reach PyPI but we can retry if the process fails.)